### DISCUSSION TITLE
Ability to rename generated page names

### DISCUSSION THREAD

**Discussion by Xerios** [sveltejs/kit#3689] [Q&A]
An issue that has persisted with our website for quite a while is our `privacy-policy.svelte` page being blocked by an adblock filter because it contains the words "privacy". 

Is it possible to rename (conditionally) generated page routes to something else?

**Comment by ignatiusmb** [comment]
No, it would be very confusing to a lot of people if this were the case. Rename the page or ask your visitors to disable ad-block.
